install/helm/open-match/subcharts/open-match-telemetry/dashboards/scale-test.json

```diff
@@ -0,0 +1,840 @@
+{
```

Comment:
I don't mind adding a grafana chart to track the emitted scale metrics but is it possible to place this file elsewhere in the helm chart - perhaps `subcharts/open-match-scale` is a better place for this schema. Otherwise users who don't wanna run the scale test will always see this empty graph when they follow the Grafana installation instructions

Reply:
To do that we need to add something which specifically selects which directories to look for dashboards in.  I'm up for that being done, but I think we should do it as a cleanup so we can make progress here.  This is a general problem, as other future extensions are going to want to add their own dashboards.

Reply:
https://github.com/googleforgames/open-match/issues/1043

Reply:
Sure. I think this is a v0.9 issue - having users seeing unexpected grafana charts is a regression comparing to what offered in v0.8. I'll look into this issue later this week.